cartoframes/viz/map.py

```diff
@@ -166,6 +168,7 @@ def __init__(self,
         self._carto_vl_path = kwargs.get('_carto_vl_path', defaults.CARTO_VL_PATH)
         self._airship_path = kwargs.get('_airship_path', None)
         self._htmlMap = HTMLMap()
+        self._publisher = None
```

Comment:
would it make sense to initialize the publisher here?

Reply:
I am not really sure, but probably yes. It could keep clear other methods. Let me check it